sftp/1.3/modules/ROOT/pages/sftp-list.adoc

```diff
@@ -1,15 +1,34 @@
 = List Files Using the SFTP Connector - Mule 4
 :page-aliases: connectors::sftp/sftp-list.adoc
 
-The `list` operation returns a list of messages, in which each message represents any file or folder found within the directory path (`directoryPath`).
+Anypoint Connector for SFTP (SFTP Connector) List operation returns a list of messages, in which each message represents any file or folder found within the directory path:
```

Comment:
Suggestion for this sentence:
...List operation returns a list of messages, and each of these represents any file or folder found within the directory path.